backend/services/src/main/java/com/crimealert/services/PostService.java

```diff
@@ -241,7 +264,8 @@ public Document[] listPosts() {
 	        System.err.println("An error occurred while attempting to run a command: " + ce);
 	        throw ce;
 	    }
-	    return new Document[1];
+		
+		return posts;
```

Comment:
Do we return sorted by latest created posts or do we sort in UI?

Reply:
we can add a filter parameter if sort if true or not
if asking for sorted list yes
else as we retrieve we can return - we add it to future backlog